data/json/professions.json

```diff
@@ -440,7 +440,7 @@
     "type": "profession",
     "ident": "sheltered_survivor",
     "name": "Sheltered Survivor",
-    "description": "At the start of the Cataclysm, you hunkered down in a bomb shelter.  Now, it is winter, and you hope the rag-tag collection of skills you learned from all those books can help you survive.",
+    "description": "At the start of the Cataclysm, you hunkered down in a bomb shelter.  You've spent the past months eating canned food and reading books.  Now it is winter, and you hope the rag-tag collection of skills you've picked up will keep you alive.",
```

Comment:
Could I suggest changing this to refer to learning those skills from reading books and practice? It's a little silly (in game and for this description) that you can learn skills simply by reading books with no practice.